File: src/main/java/com/dv/telegram/GoogleSheetBotData.java
package com.dv.telegram;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.List;

@Data
@AllArgsConstructor
public class GoogleSheetBotData {
    private final List<WikiPageData> pages;
    private final List<CityChatData> cityChats;
    private final List<WikiBotCommandData> commands;
}

File: src/main/kotlin/com/dv/telegram/GoogleSheetBotData.kt
package com.dv.telegram

import com.dv.telegram.data.CityChatData
import com.dv.telegram.data.CountryChatData
import com.dv.telegram.data.WikiBotCommandData
import com.dv.telegram.data.WikiPageData

@Deprecated("Use [GoogleSheetBotTabsData] instead.")
data class GoogleSheetBotData(
    val pages: List<WikiPageData>,
    val cityChats: List<CityChatData>,
    val countryChats: List<CountryChatData>,
    val commands: List<WikiBotCommandData>
)
